components/script/dom/servohtmlparser.rs

```diff
@@ -380,17 +375,15 @@ impl ServoHTMLParser {
             document: JS::from_ref(document),
         };
 
-        let tb = TreeBuilder::new(sink, TreeBuilderOpts {
-            ignore_missing_rules: true,
-            .. Default::default()
+        let html5ever_parser = parse_document(sink, Default::default()).from_bytes(BytesOpts {
+            // FIXME: get this from Hyper
```

Comment:
See https://github.com/SimonSapin/kuchiki/blob/2e6d261cb39e555afad54d5cccdc9170df747cf9/src/parser.rs#L71-L75 for how to get this from an `hyper::client::Response`.